src/plugins/mer/merbuildenginedetailswidget.ui

```diff
@@ -147,7 +147,7 @@
             <item>
              <widget class="QLabel" name="sshPortInfoLabel">
               <property name="sizePolicy">
-               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
```

Comment:
Intentional?

Reply:
I reverted it, but added spacer, because I think two info labels should be close.

Reply:
Both work for me, I just wanted to ensure it is not a garbage added by designer unintentionally :)